components/list/list-item-mixin.js

```diff
@@ -147,12 +147,16 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			:host(:last-of-type[_separators="between"]) [slot="control-container"]::after,
 			:host([_separators="none"]) [slot="control-container"]::before,
 			:host([_separators="none"]) [slot="control-container"]::after,
-			:host([selectable][_hovering]:not([selection-disabled])) [slot="control-container"]::before,
-			:host([selectable][_hovering]:not([selection-disabled])) [slot="control-container"]::after,
```

Comment:
The `_hovering` property still exists right? Why do we need to change all the uses of this property. Or was the intention to remove it altogether?

Reply:
It still exists but it is set to true even if hovering over a secondary action, this means styles where being applied to both the checkbox and the list item even though neither selection nor primary action where being hovered.